src/components/Home.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { Redirect } from 'react-router'
+
+
+
+export const Home = ({ ticket }) => {
+    if (ticket && ticket.originId) {
+        return <Redirect to={`/tickets/${ticket.originId}`} />
+    }
+    return <Redirect to={'/search'} />
+}
+
+export const mapStateToProps = (state) => {
+    // this gross thing finds all actual tickets, omitting the refresh metadata, and then picks the most recently visited one out
```

Comment:
Eventually, I'd prefer to keep track of this by dispatching something to a reducer when the user loads a new incident, since this implementation keeps us from doing things like refreshing incidents in the background every few minutes.